packages/ember-data/lib/system/model/model.js

```diff
@@ -901,4 +902,19 @@ Model.reopenClass({
   modelName: null
 });
 
+if (Ember.platform.hasPropertyAccessors) {
+  var errorDeprecationShown = false;
+
+  Ember.defineProperty(Model.prototype, 'error', {
+    enumerable: false,
+    get: function() {
+      if (!errorDeprecationShown) {
+        Ember.deprecate('DS.Model#error has been deprecated please use adapterError instead');
+        errorDeprecationShown = true;
+      }
+      return Ember.get(this, 'adapterError');
+    }
+  });
+}
```

Comment:
right, but now that I am thinking, if someone defined `error` attribute on the model, the deprecation will not work anyway. Not sure what to do about it. Maybe it is double, but I have no idea how to define a property that can be overridden but will still run some code on get...